source/change-streams/change-streams.rst

```diff
@@ -1007,3 +1069,5 @@ Changelog
 +------------+------------------------------------------------------------+
 | 2022-05-17 | Added ``wallTime`` to ``ChangeStreamDocument``.            |
 +------------+------------------------------------------------------------+
+| 2022-05-19 | Support new change stream events for C2C replication.      |
```

Comment:
```suggestion
| 2022-05-19 | Support new change stream events with showExpandedEvents.      |
```
I'm not sure we even need to mention C2C replication. We're really just supporting the `showExpandedEvents` flag.

Reply:
Good call.  Done!